content/en/tracing/setup/php.md

```diff
@@ -100,8 +100,10 @@ The PHP tracer can be configured using environment variables.
 For Apache with php-fpm, use the `env` directory in your `www.conf` configuration file to configure the php tracer, for example:
 
 ```
-env[DD_AGENT_HOST] = $FROM_HOST_ENV
-env[DD_TRACE_DEBUG] = true
+; Both actual values (e.g. localhost) and references
```

Comment:
I had a little trouble understanding this sentence at first glance. Perhaps we can annotate each example separately?

```
; Example of passing the environment variable SOME_ENV
; to the PHP process as DD_AGENT_HOST
env[DD_AGENT_HOST] = $SOME_ENV
; Example of passing the value 'my-app' to the PHP
; process as DD_SERVICE
env[DD_SERVICE] = my-app
```